src/deepsparse/server/config.py

```diff
@@ -49,6 +49,63 @@ class ImageSizesConfig(BaseModel):
     )
 
 
+class SystemLoggingGroup(BaseModel):
+    """
+    Holds the configuration for a single system logging group.
+    """
+
+    enable: bool = Field(
+        default=False, description="Whether to enable the system logging group"
+    )
+
+    target_loggers: Optional[List[str]] = Field(
+        default=None,
+        description="The list of target loggers to log to. "
+        "If None, logs to all the available loggers",
+    )
+
+
+class SystemLoggingConfig(BaseModel):
+    """
+    Holds the configuration for the system logging
+    """
+
+    # Global System Logging
+    enable: Optional[bool] = Field(
+        default=True, description="Whether to enable system logging"
+    )
+    target_loggers: Optional[List[str]] = Field(
+        default=None,
+        description="The list of loggers to log to. If None, "
+        "logs to all the available loggers.Will "
+        "be overriden by the target_loggers specified "
+        "in the individual system logging groups.",
+    )
+
+    # System Logging Groups
+    resource_utilization: Optional[SystemLoggingGroup] = Field(
+        default=None,
```

Comment:
Could probably make them not optional and make default explicitly enable=False for all these groups:
```suggestion
    resource_utilization: SystemLoggingGroup = Field(
        default=SystemLoggingGroup(enable=False),
```

Reply:
+1, will make handling on our end more straightforward.  Additionally, why do we default system logging to True, but then disable all of the groups?

Reply:
To enforce some defaults (I imagined prediction_latency would be a default). For the sake of this PR I would refrain from setting any defaults, they can come later in a form of a very short PR.